.github/workflows/build-java-multi.yaml

```diff
@@ -202,8 +202,25 @@ jobs:
         with:
           registries: ${{ needs.init.outputs.ecr-registry-code }}
 
-      - name: Build Docker Image
+      - name: Check if Docker image tag exists
         if: ${{ github.workflow != 'pr' && steps.ecr-info.outputs.module_ecr != 'null' }}
+        id: check-tag
+        env:
+          ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
+          IMAGE_TAG: ${{ needs.init.outputs.image-tag }}
+        run: |
+          # Extract the repository name using jq
```

Comment:
Would be perfect if you created a new action to check for the docker image name/tag and then apply it everywhere, even for the single pipeline